api/src/main/kotlin/com/okeicalm/simpleJournalEntry/entity/Journal.kt

```diff
@@ -13,7 +13,7 @@ data class Journal(
     constructor(journal: Journals, journalEntries: List<JournalEntries>?) : this(
         journal.id,
         journal.incurredOn,
-//        journalEntries?.map { JournalEntry(it) },
-        null,
+        journalEntries?.map { JournalEntry(it) },
+//        null,
```

Comment:
Do you need this comment out? If unnecessary, please remove this line.

Reply:
Correction.